app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -70,7 +70,14 @@ keywords:
 
 $new-international-funding$
 
-Citizens of every country in the world can apply to train to teach in England.
+Citizens of every country in the world can apply to train to teach in England. When you plan your training, bear in mind that:
+
+- international candidates can expect to pay between £9,250 and £32,000, plus
+living costs, to train (depending on your immigration status and the subject you train to teach, you may get some financial support)
+- training lasts one academic year full-time, or longer part-time
+- you can apply for paid teaching jobs as soon as you successfully complete your training
+- you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher
+- you can update, extend or switch your visa to live and work in the UK long-term
```

Comment:
@laurahermionetennant92 can anyone from any country do this and be able to live and work in the UK long-term?

Reply:
Yes